apps/vm-lang/main.cpp

```diff
@@ -336,5 +336,10 @@ int main(int argc, char **argv)
     state_map.SaveToFile(data_path.c_str());
   }
 
+  if (!success)
```

Comment:
This code will never be called

Reply:
I don't even know how this bit of code got here. I suspect it has to be artifact of weird merge of master branch in to resurrected branch of original PR.

Btw, there are other weird pieces of code similar to this one above (lines 327-331) which is not supposed here either.